test/parallel/test-domain-from-timer.js

```diff
@@ -12,7 +12,7 @@ setTimeout(function() {
   d.run(function() {
     process.nextTick(function() {
       console.trace('in nexttick', process.domain === d);
-      assert.equal(process.domain, d);
+      assert.strictEqual(process.domain, d);
     });
   });
-});
+}, 0);
```

Comment:
It should probably also be a duration of `1`. Duration of 0 gets changed to 1 internally anyways.